docs/Daedalus-Installer-Update-Procedure.md

```diff
@@ -0,0 +1,267 @@
+# Daedalus Installer Update Procedure
+
+After the installers are built in CI, they need to be copied to the
+installer updates S3 bucket, and an update proposal needs to be
+submitted to the cardano-sl network.
+
+This is how wallet clients will self-update.
+
+**New!** There have recently been some changes and improvements to
+these commands from [DEVOPS-868][].
+
+## Running the scripts
```

Comment:
Should there be a prerequisits section here? What do I need to actually perform these steps? And if I don't have the prerequisits whom should I contact to get them? SSH key auth, ...